release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -698,17 +704,52 @@ submodule openconfig-bgp-common {
     description
       "Parameters controlling usage of  of DMZ Link-Bandwidth
       extended community in pultipath RIB/FIB formation";
+
     leaf enabled {
       type boolean;
       description
-      "When set to TRUE, BGP multiplepath shall distributed traffic
-      load among contributing routes proportionally to value of
-      Local Administrator subfield of link-bandwidth extended
-      community [draft-ietf-idr-link-bandwidth-07].
-      This leaf has no effect if BGP multi-path is disabled or
-      if maximum-path attribute of BGP multi-path value is set
-      to 1";
+        "When set to TRUE, BGP multiplepath shall distributed traffic
+        load among contributing routes proportionally to value of
+        Local Administrator subfield of link-bandwidth extended
+        community [draft-ietf-idr-link-bandwidth-07].
+        This leaf has no effect if BGP multi-path is disabled or
+        if maximum-path attribute of BGP multi-path value is set
+        to 1";
+    }
+
+    leaf cumulative {
+      type boolean;
+      when "../enabled = 'TRUE'";
+      description
+        "When set to TRUE, and sending of link-bandwidth community values
+        is enabled, the system will sum the bgp link-bandwidth community
+        values for equal cost paths as defined in draft-ietf-bess-ebgp-dmz.";
+      reference
+        "https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/";
+    }
+
+    leaf non-transitive {
+      type boolean;
+      when "../enabled = 'TRUE'";
+      description
+        "When set to TRUE and sending of link-bandwidth community values is
+        enabled, the system will send the link-bandwidth community to eBGP
+        peers.";
+      reference
+        "https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/";
+    }
+
+    leaf equal {
```

Comment:
I'd suggest including the word "divide" in the name in some form.

Reply:
average? it is average is not?

Reply:
"average" is effectively what it is.  That said, there's some desire to have symmetry in the cited draft.  I'd suggest either picking that as a name and working with the authors to incorporate it in their description, or stick to something in the draft like divide.

Note that Juniper has chosen divide-equal in its policy language.

Reply:
I'll use divide since that is in the draft.